iroh-p2p/src/rpc.rs

```diff
@@ -255,15 +256,41 @@ impl RpcP2p for P2p {
     }
 
     #[tracing::instrument(skip(self, req))]
-    async fn peer_connect(&self, req: ConnectRequest) -> Result<ConnectResponse> {
+    /// peer_connect_by_peer_id will attempt to find the peer on the DHT
```

Comment:
Please do not prefix docs with function names, just write a regular sentence :)

Reply:
my golang is showing 😆

Reply:
old habits die hard 😄